code/modules/jobs/job_types/roguetown/mercenaries/classes/rumaclan.dm

```diff
@@ -0,0 +1,85 @@
+			gloves = /obj/item/clothing/gloves/roguetown/eastgloves2
+			backr = /obj/item/storage/backpack/rogue/satchel
+			backpack_contents = list(/obj/item/roguekey/mercenary, /obj/item/flashlight/flare/torch/lantern)
+			H.mind.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/shields, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+			H.change_stat("strength", 2)
+			H.change_stat("endurance", 3)
+			H.change_stat("constitution", 3)
+			H.change_stat("perception", 1)
+			H.change_stat("speed", -1)
+
+		if("Ishu")
+			H.set_blindness(0)
+			to_chat(H, span_warning("You are an archer. Pretty good in the art of \"pelting until enemy death.\" - You would gladly take up most jobs for money, or a chance to shoot loose."))
+			belt = /obj/item/storage/belt/rogue/leather
+			beltr = /obj/item/quiver/arrows
+			beltl = /obj/item/flashlight/flare/torch/lantern
+			shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/eastshirt2
+			cloak = /obj/item/clothing/cloak/eastcloak1
+			armor = /obj/item/clothing/suit/roguetown/shirt/undershirt/easttats
+			pants = /obj/item/clothing/under/roguetown/trou/eastpants2
+			shoes = /obj/item/clothing/shoes/roguetown/armor/rumaclan
+			gloves = /obj/item/clothing/gloves/roguetown/eastgloves2
+			backl = /obj/item/gun/ballistic/revolver/grenadelauncher/bow/recurve
+			backr = /obj/item/storage/backpack/rogue/satchel
+			backpack_contents = list(/obj/item/roguekey/mercenary, /obj/item/storage/belt/rogue/pouch/coins/poor, /obj/item/rogueweapon/huntingknife/idagger)
+			H.mind.adjust_skillrank(/datum/skill/combat/bows, 5, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/knives, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/athletics, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/tracking, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/swimming, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/craft/carpentry, 2, TRUE)
+			H.change_stat("constitution", -1)
+			H.change_stat("endurance", 2)
+			H.change_stat("perception", 2)
+			H.change_stat("speed", 3)
+			H.change_stat("strength", -1)
```

Comment:
Likewise /this/ is weighted to 7 for a merc class. Should be 9, unless it's a special case.

Reply:
Not that I mind classes have higher or lower stats, but that generally invites some guy 2+ weeks later to notice, point it out, and PR a "correction".